llvm/test/Transforms/InstCombine/and.ll

```diff
@@ -1818,7 +1818,7 @@ define i8 @not_ashr_bitwidth_mask_use2(i8 %x, i8 %y) {
 ; CHECK-NEXT:    [[ISNOTNEG:%.*]] = icmp sgt i8 [[X:%.*]], -1
```

Comment:
Adjust the comment above saying "negative test - extra use". It's no longer a negative test.

Reply:
Let me know if it is not correct?